src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -206,6 +206,10 @@ public function getBundles()
      */
     public function getBundle($name, $first = true)
     {
+        if (false === $first) {
```

Comment:
at least `locateResource` is calling it with false.. should we get rid of $first there too? I believe it serves the same purpose.

Reply:
Correct, fixed